libraries/Microsoft.Bot.Builder.Core/Adapters/ConsoleAdapter.cs

```diff
@@ -50,11 +50,11 @@ public new ConsoleAdapter Use(IMiddleware middleware)
 
         public override async Task<ResourceResponse[]> SendActivities(ITurnContext context, Activity[] activities)
         {
-            List<ResourceResponse> responses = new List<ResourceResponse>();
+            var responses = new ResourceResponse[activities.Length];
```

Comment:
Null check. This will throw if activities is null. 

Should also check (and throw) if the length is 0.

Reply:
✔️